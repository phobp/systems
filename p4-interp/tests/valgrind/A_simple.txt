==9559== Memcheck, a memory error detector
==9559== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==9559== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==9559== Command: ../y86 -e inputs/simple.o
==9559== 
Beginning execution at 0x0100
Memory write to 0x0ef8: 0x113
Memory write to 0x0400: 0x7fffffffffffffff
Y86 CPU state:
  %rip: 0000000000000000   flags: Z0 S0 O0     HLT
  %rax: 7fffffffffffffff    %rcx: 0000000000000001
  %rdx: 0000000000000000    %rbx: 0000000000000000
  %rsp: 0000000000000f00    %rbp: 0000000000000000
  %rsi: 0000000000000000    %rdi: 0000000000000000
   %r8: 0000000000000000     %r9: 0000000000000400
  %r10: 0000000000000000    %r11: 0000000000000000
  %r12: 0000000000000000    %r13: 0000000000000000
  %r14: 0000000000000000
Total execution count: 15

==9559== 
==9559== HEAP SUMMARY:
==9559==     in use at exit: 552 bytes in 1 blocks
==9559==   total heap usage: 5 allocs, 4 frees, 21,176 bytes allocated
==9559== 
==9559== LEAK SUMMARY:
==9559==    definitely lost: 0 bytes in 0 blocks
==9559==    indirectly lost: 0 bytes in 0 blocks
==9559==      possibly lost: 0 bytes in 0 blocks
==9559==    still reachable: 552 bytes in 1 blocks
==9559==         suppressed: 0 bytes in 0 blocks
==9559== Rerun with --leak-check=full to see details of leaked memory
==9559== 
==9559== For counts of detected and suppressed errors, rerun with: -v
==9559== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
