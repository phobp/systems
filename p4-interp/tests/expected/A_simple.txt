Beginning execution at 0x0100
Memory write to 0x0ef8: 0x113
Memory write to 0x0400: 0x7fffffffffffffff
Y86 CPU state:
  %rip: 0000000000000000   flags: Z0 S0 O0     HLT
  %rax: 7fffffffffffffff    %rcx: 0000000000000001
  %rdx: 0000000000000000    %rbx: 0000000000000000
  %rsp: 0000000000000f00    %rbp: 0000000000000000
  %rsi: 0000000000000000    %rdi: 0000000000000000
   %r8: 0000000000000000     %r9: 0000000000000400
  %r10: 0000000000000000    %r11: 0000000000000000
  %r12: 0000000000000000    %r13: 0000000000000000
  %r14: 0000000000000000
Total execution count: 15

